oka/chap2/knock10.py

```diff
@@ -0,0 +1,9 @@
+import subprocess
+
+
+path = "C:\\Users\\16t215\\Desktop\\hightemp.txt"
+with open(path, encoding="utf-8") as f:
+    s = f.readlines()
+    print(len(s))
+queri = ["wc", "-l", path]
```

Comment:
`queri`って`query`のこと？

```suggestion
query = ["wc", "-l", path]
```